content/docs/apps/continuous-deployment.md

```diff
@@ -28,16 +28,16 @@ You can provision a deployer account with permission to deploy to a single space
 1. Create a deployer account service instance
 
     ```bash
-    $ cf create-service deployer-account deployer-account my-deployer-account
+    $ cf create-service cloud-gov-service-account space-deployer my-service-account
```

Comment:
I'd make it `my-deployer-account` to encourage people to choose names relevant to them for specific instance that they create.